--- src/basic/linux/if_arp.h    2021-11-28 17:19:23.639402182 +0000
+++ src/basic/linux/if_arp.h    2021-11-28 17:19:47.435466956 +0000
@@ -54,6 +54,7 @@
 #define ARPHRD_X25     271             /* CCITT X.25                   */
 #define ARPHRD_HWX25   272             /* Boards with X.25 in firmware */
 #define ARPHRD_CAN     280             /* Controller Area Network      */
+#define ARPHRD_MCTP    290
 #define ARPHRD_PPP     512
 #define ARPHRD_CISCO   513             /* Cisco HDLC                   */
 #define ARPHRD_HDLC    ARPHRD_CISCO
